core/src/main/java/com/graphhopper/routing/util/spatialrules/SpatialRuleLookupArray.java

```diff
@@ -53,15 +53,34 @@
      * @param exact      if exact it will also perform a polygon contains for border tiles, might fail for small holes
      *                   in the Polygon that are not represented in the tile array.
      */
-    public SpatialRuleLookupArray(BBox bounds, double resolution, boolean exact) {
+    SpatialRuleLookupArray(BBox bounds, double resolution, boolean exact) {
+        if (bounds == null)
+            throw new IllegalArgumentException("BBox cannot be null");
+        if (resolution < 1e-100)
```

Comment:
I am perfectly fine with the limit, I was just wondering why it is that number :).